README.md

```diff
@@ -4,10 +4,42 @@ Embed Bluesky comments on your website easily.
 
 **[Write up and demo here](https://coryzue.com/writing/bluesky-comments).**
 
+## React Installation
 
-## Installation via CDNs (easiest)
+To use this library in a React project, first install the library:
 
-There are a few ways to set up the library on your website.
+```bash
+npm install bluesky-comments
+```
+
+Then import it (and the CSS) in your React app/page/component:
+
+```tsx
+import 'bluesky-comments/bluesky-comments.css'
```

Comment:
I found (in my test vite app at least) that I still had to do this. If that's not expected, any idea under what circumstances that might be true?

Reply:
If using in a React app this shouldn't be needed. Take a look at the `index.tsx` file in `/dev` -- no need to import the css since the component itself is importing.

If you are using this in the non JSX way, then yes, you will need to import the CSS.

Reply:
Weird, maybe an issue with my dev setup then? What I did:

1. Run `npm run build`
2. In a separate project, add this to package.json:
```json
  "dependencies": {
    "react": "^18.3.1",
    "react-dom": "^18.3.1",
    "bluesky-comments": "file:../../bluesky-comments"
  },
  ```
  3. Import and run like this:
  
  ```jsx
// import 'bluesky-comments/bluesky-comments.css'
import {useContext, useState} from 'react'
import './App.css'
import { BlueskyComments } from "bluesky-comments";


function App() {
  const { token, user, isAuthenticated, isLoading } = useContext(AuthContext);
  return (
    <>
        <BlueskyComments
           uri="https://bsky.app/profile/coryzue.com/post/3lbrko5zsgk24"
            />

    </>
  )
}

export default App
```
If I leave that css import commented out the styles don't work, but if I add it they do.